core/servers/chat/mrc_multiplexer.js

```diff
@@ -336,6 +335,13 @@ exports.getModule = class MrcModule extends ServerModule {
         //     return;
         // }
 
+        // Make sure to_user is always uppercase
+        try {
+            let to_user_uc = to_user.toUpperCase();
+            to_user = to_user_uc;
```

Comment:
This could just be `to_user = to_user.toUpperCase()`, no temp var. Or without the try:
```javascript
to_user = (to_user || '').toUpperCase()
```